admission/Makefile

```
@@ -6,6 +6,10 @@ GRAFANA_OPERATOR_V4_VERSION = 4.10.1
 GRAFANA_OPERATOR_VERSION = 5.8.1
 ENVTEST_K8S_VERSION = 1.27.1
 
+# Specify envtest branch according to controller-runtime version
+CONTROLLER_RUNTIME_VERSION := $(shell awk '/sigs\.k8s\.io\/controller-runtime/ {print substr($$2, 2)}' go.mod)
```

```suggestion
CONTROLLER_RUNTIME_VERSION := $(shell awk '$$1 == "sigs.k8s.io/controller-runtime" {print substr($$2, 2)}' go.mod)
```

It would be nice to check module name strictly.

Other two modifications too.